Cargo.toml

```diff
@@ -41,12 +46,15 @@ zbus = { version = "3.10.0" }
 nix = "0.26.2"
 rand = "0.8.5"
 sha2 = "0.10.6"
-netlink-packet-route = "0.13"
-netlink-packet-core = "0.4.2"
+netlink-packet-route = "0.15"
+netlink-packet-core = "0.5.0"
+netlink-packet-wireguard = { git = "https://github.com/rust-netlink/netlink-packet-wireguard", version = "0.2.2" }
```

Comment:
is there a specific thing you need from the gihub version, is it not pushed to crates.io?

Reply:
I noticed that the repo [repo](https://github.com/rust-netlink/netlink-packet-wireguard/blob/main/Cargo.toml) and the [crate](https://crates.io/crates/netlink-packet-wireguard) have gotten out of sync.

My thought was to track the repo  so that the libraries stay compatible.
Do you think this is a problem from a security perspective?

Reply:
Not really, it will require distros that use vendored sources to update the cargo.conf to include the new location. Not a big deal just annoying. We know one of the maintainers so I can ask them to publish it on crates.io, likely some oversight.

Reply:
Very nice. I will update it afterwards :+1: